macros/models/_project/oneinch/oneinch_project_swaps_macro.sql

```diff
@@ -0,0 +1,162 @@
+{% macro 
+    oneinch_project_swaps_macro(
+        blockchain
+        , date_from = '2024-01-01'
```

Comment:
from my understanding in conversation, initial merge will only read 2024 data. the data will be further tested, then look to do more history in follow up PR.

Reply:
no action on this comment, just clarifying.

Reply:
yes!